### PR TITLE
fix(x11/tuxpaint): add missing desktop entry

### LINKED ISSUE #21890: [Bug]: tuxpaint does not install `.desktop` file, the program is not listed in xfce's applications menu
### Problem description

A title describes all of it.

### What steps will reproduce the bug?

`pkg install tuxpaint`

### What is the expected behavior?

It should install `.desktop` file and the program should be visible in `Applications` menu.

### System information

```shell
Irrelevant.
```

### PR DESCRIPTION
**Pull request by twaik**
Fixes #21890

I will merge it in 72 hours if no one minds.
Or anyone else can merge it earlier.

### CODE REVIEW

File: x11-packages/tuxpaint/build.sh [hunk 1]
```diff
@@ -33,8 +35,12 @@ termux_step_pre_configure() {
...
 	TERMUX_PKG_DEPENDS+=", tuxpaint-data"
 
-	local _PREFIX_FOR_BUILD=$TERMUX_PKG_HOSTBUILD_DIR/prefix
-	export PATH=$_PREFIX_FOR_BUILD/bin:$PATH
+	local _PREFIX_FOR_BUILD="$TERMUX_PKG_HOSTBUILD_DIR/prefix"
+	export PATH="$_PREFIX_FOR_BUILD/bin:$PATH"
+	export XDG_DATA_HOME="$TERMUX_PREFIX/share" XDG_DATA_DIRS="$TERMUX_PREFIX" XDG_CURRENT_DESKTOP="X-Generic"
+	
+	# Disabling gtk-update-icon-cache
+	ln -s /usr/bin/true "$_PREFIX_FOR_BUILD/bin/gtk-update-icon-cache" ||:
```
Comment (TomJo2000) [inline x11-packages/tuxpaint/build.sh:43]: Don't think I've seen that one before.
Why exactly are we disabling `gtk-update-icon-cache` here?

Comment (twaik) [reply x11-packages/tuxpaint/build.sh:43]: I am not really sure. It is disabled everywhere during cross-build.
[1](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/azpainter/install.sh.in.patch) [2](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/xfdesktop/dont-update-icon-cache.patch) [3](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/gtk3/0002-dont-update-icon-cache.patch) [4](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/xfce4-panel/dont-update-icon-cache.patch) [5](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/xfce4-session/dont-update-icon-cache.patch)

Comment (twaik) [reply x11-packages/tuxpaint/build.sh:43]: As you can see something here breaks build so I will investigate it more.

Comment (TomJo2000) [reply x11-packages/tuxpaint/build.sh:43]: Guessing it just doesn't apply to Termux.
Oh well.

**Pull request merged**